internals/scripts/build.js

```diff
@@ -0,0 +1,11 @@
+// @ts-check
+const execSync = require('child_process').execSync
+const buildSha = require('child_process').execSync('git rev-parse HEAD').toString().trim()
+const buildDatetime = Date.now().toString()
+
+process.env.REACT_APP_BUILD_DATETIME = buildDatetime
+process.env.REACT_APP_BUILD_SHA = buildSha
```

Comment:
Why are these 2 lines needed? Seems to be like they are being passed inline in the execSync command.

Reply:
oh yes it is leftover. thx